src/Stateless/Stateless.csproj

```diff
@@ -1,37 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Description>Create state machines and lightweight state machine-based workflows directly in .NET code</Description>
     <AssemblyTitle>Stateless</AssemblyTitle>
+    <AssemblyName>Stateless</AssemblyName>
+    <Product>Stateless</Product>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <Description>Create state machines and lightweight state machine-based workflows directly in .NET code</Description>
+    <Copyright>Copyright © Stateless Contributors 2009-2020</Copyright>
     <NeutralLanguage>en-US</NeutralLanguage>
-    <VersionPrefix>4.2.0</VersionPrefix>
+    <VersionPrefix>4.3.0</VersionPrefix>
+    <Version>4.3.0</Version>
     <Authors>Stateless Contributors</Authors>
-    <TargetFrameworks>netstandard1.0;net45;net40</TargetFrameworks>
```

Comment:
there is no need to remove the older supported frameworks and netstandard, as it works perfectly fine with Stateless

Reply:
netstandard 2 can be consumed from [net461 and above](https://docs.microsoft.com/en-us/dotnet/standard/net-standard). why support old frameworks that will be dead/unsupported soon (net50 will accelerate this)

Reply:
I won't argue with you any further as I made my point clear

Reply:
let's be constructive: a lot of us just need netstandard2.0 support (with or without net45/netstandard1.0). there are several other PRs waiting (#322, #320, #333,), however i don't see any progress here...

referring to your needs #320 has the best chances (all target frameworks + netstandard2.0)
https://github.com/dotnet-state-machine/stateless/pull/320/files#diff-0

how do we get this forward? (@HenningNT , @nblumhardt )

Reply:
Hi! Is there any reason to drop other targets to add `netstandard2.0`? Seems like a loss of functionality for no benefit, at this point - why not just add one more target framework? (Apologies if I'm missing something - thanks for looping me in!)

Reply:
done: net462 added

Reply:
done: netstandard2.0 added

Reply:
Hi @nblumhardt, thanks for jumping in :)
As you can see there have been several PRs regarding netstandard2.0 support, lot's of folks want it. Maybe this or any of the other PRs (but not the netststandard2.1 one) can be merged, it's a small thing which helps us move forward.